packages/ng/simple-select/input/select-input.component.html

```diff
@@ -1,7 +1,9 @@
 <div class="simpleSelect-field">
-	@if (prefix()) {
-		<span class="simpleSelect-field-prefix"><ng-container *luPortal="prefix()" /></span>
-	}
+	<span class="simpleSelect-field-prefix">
```

Comment:
This means that even without a prefix, we'd have this span, wouldn't cause issues?